src/main/java/cpw/mods/ironchest/plugins/nei/NEIConfig.java

```diff
@@ -0,0 +1,24 @@
+package cpw.mods.ironchest.plugins.nei;
+
+import codechicken.nei.DefaultBookmarkContainerHandler;
+import codechicken.nei.api.API;
+import codechicken.nei.api.IConfigureNEI;
+import cpw.mods.ironchest.client.GUIChest;
+
+public class NEIConfig implements IConfigureNEI {
+
+    @Override
+    public void loadConfig() {
+        API.registerBookmarkContainerHandler(GUIChest.class, new DefaultBookmarkContainerHandler());
+    }
+
+    @Override
+    public String getName() {
+        return "Iron Chests";
+    }
+
+    @Override
+    public String getVersion() {
+        return "${version}";
```

Comment:
This method should return proper version. In most cases version of mod is used (see different [implementations](https://github.com/search?q=org%3AGTNewHorizons%20IConfigureNEI&type=code))

Reply:
done